libp2p/protocol_muxer/multiselect.py

```diff
@@ -46,7 +46,10 @@ def add_handler(self, protocol: TProtocol, handler: StreamHandlerFn) -> None:
         # Read and respond to commands until a valid protocol ID is sent
         while True:
             # Read message
-            command = await communicator.read()
+            try:
+                command = await communicator.read()
+            except MultiselectCommunicatorError as error:
+                raise MultiselectError(str(error))
```

Comment:
```suggestion
                raise MultiselectError(error)
```

i think we can do this, right? (just less code...)